internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-manifest.yml

```diff
@@ -60,6 +60,11 @@ environments:
         path: /
         grace_period: 30s
       deregistration_delay: 30s
+    secrets:                      
```

Comment:
Can we add the secret to a sidecar and a logging's [secret options](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#logging-secretOptions) just to make sure it works well for those two places as well?